src/DssSpell.sol

```diff
@@ -75,334 +60,240 @@ contract DssSpellAction is DssAction {
+            l2Tokens: l2Tokens,
+            // Set parameter cfg.maxWithdraws: (type(uint256).max for each token)
+            maxWithdraws: maxWithdrawals,
+            // Set parameter cfg.minGasLimit: (500,000)
+            minGasLimit: 500_000,
+            // Set parameter cfg.govRelayCLKey: (chainlog key for govRelay -> BASE_GOV_RELAY)
+            govRelayCLKey: "BASE_GOV_RELAY",
+            // Set parameter cfg.escrowCLKey: (chainlog key for Escrow -> BASE_ESCROW)
+            escrowCLKey: "BASE_ESCROW",
+            // Set parameter cfg.l1BridgeCLKey: (chainlog key for L1TokenBridge -> BASE_TOKEN_BRIDGE)
+            l1BridgeCLKey: "BASE_TOKEN_BRIDGE",
+            // Set parameter cfg.l1BridgeImpCLKey: (chainlog key for L1TokenBridgeImp -> BASE_TOKEN_BRIDGE_IMP)
+            l1BridgeImpCLKey: "BASE_TOKEN_BRIDGE_IMP"
+        });
+
+        // Init Base Token Bridge for USDS, sUSDS
+        TokenBridgeInit.initBridges(dss, l1BridgeInstance, l2BridgeInstance, bridgeCfg);
+
+
+        // ---------- Init Allocator System for Spark Subdao Proxy ----------
+        // Forum: https://forum.sky.money/t/spell-contents-2024-10-31/25421/
+        //
+        // Init ALLOCATOR-SPARK-A ilk on vat, jug and spotter
+        // Set duty on jug to 5.2%
+        // Set line on vat
+        // Increase Global Line on vat
+        // Setup AutoLine for ALLOCATOR-SPARK-A:
+        // line: 10_000_000
+        // gap: 10_000_000
+        // ttl: 86_400 seconds
+        // Set spotter.pip for ALLOCATOR-SPARK-A to AllocatorOracle contract
+        // Set spotter.mat for ALLOCATOR-SPARK-A to RAY
+        // poke ALLOCATOR-SPARK-A (spotter.poke)
+        // Add AllocatorBuffer address to AllocatorRegistry
+        // Initiate the allocator vault by calling vat.slip & vat.grab
+        // Set jug on AllocatorVault
+        // Allow vault to pull funds from the buffer by giving max USDS approval
+        // Set the allocator proxy as the ALLOCATOR-SPARK-A ilk admin instead of the Pause Proxy on AllocatorRoles
+        // Move ownership of AllocatorVault & AllocatorBuffer to AllocatorProxy (SparkProxy)
+        // Add Allocator contracts to chainlog (ALLOCATOR_ROLES, ALLOCATOR_REGISTRY, ALLOCATOR_SPARK_A_VAULT, ALLOCATOR_SPARK_A_BUFFER, PIP_ALLOCATOR_SPARK_A)
+        // Add ALLOCATOR-SPARK-A ilk to IlkRegistry
+
+        // Set sharedInstance with the following parameters:
+        AllocatorSharedInstance memory allocatorSharedInstance = AllocatorSharedInstance({
+            // Set parameter sharedInstance.oracle: (Allocator Oracle: 0xc7B91C401C02B73CBdF424dFaaa60950d5040dB7)
+            oracle:   PIP_ALLOCATOR,
+            // Set parameter sharedInstance.roles: (AllocatorRoles: 0x9A865A710399cea85dbD9144b7a09C889e94E803)
+            roles:    ALLOCATOR_ROLES,
+            // Set parameter sharedInstance.registry: (AllocatorRegistry: 0xCdCFA95343DA7821fdD01dc4d0AeDA958051bB3B)
+            registry: ALLOCATOR_REGISTRY
+        });
+
```

Comment:
`Init Allocator ILK for Spark Subdao by calling AllocatorInit.initIlk using the following parameters:` is missing